src/common/components/cookie-notification/index.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+
+// importing styles directly from cookie-policy module
+import styles from '../../../../node_modules/cookie-policy/build/css/cookie-policy.css';
+
+// Quick React-friendly reimplementation of cookie-policy script:
+// https://github.com/canonical-webteam/cookie-policy/blob/master/src/js/cookie-policy.js
+// Note: it doesn't support any settings (like custom content or delay)
+export default class CookieNotification extends Component {
```

Comment:
Interesting, is there a benefit of rewriting this into a React component over using the module JS?

Reply:
cookie-policy.js doesn't work well as JS module. It's implemented in a way that only supports using it as a global script that adds itself to global `ubuntu` namespace.

I tried to import it as a module in hope that I could reuse the implementation, but it gets encapsulated in a module, so it doesn't add any global variables. And because it doesn't export anything, it's not visible outside of local module scope.
Instead of working around that and trying to make cookie-policy.js fit into quite uncommon (at least for current webteam projects) node build system it was easier and quicker to duplicate the implementation in React component.

I guess it would be possible to update cookie-policy script to allow importing it as JS module in node environment, but I didn't feel it would fit into scope of this fix.

Reply:
@anthonydillon I had a second look and in theory it wouldn't be too hard to make cookie-policy work as a module. Just a matter of adding:

```js
if (typeof module === "object" && module.exports) {
  module.exports = ubuntu.cookiePolicy;
}
```

But, it still doesn't work out of the box in BSI because of how CSS Modules obfuscate class names. cookie-policy script creates HTML using Vanilla class names, but if we include cookie-policy.css as a CSS Modules these class names are not available anymore.
So even while we could quite easily make script itself work, it would still require loading CSS not as a CSS Module which would be something we didn't need to do in BSI before.

Probably in the future once build functionality is moved more and more into unified snapcraft.io maybe we will need to get rid of CSS module loading to have better and easier reusability of Vanilla CSS in BSI.